Signal/src/Signal-Bridging-Header.h

```diff
@@ -10,9 +10,9 @@
 #import "ConversationViewItem.h"
 #import "DateUtil.h"
```

Comment:
I haven't tested yet, but I'm assuming the sound used for the fallback push notifications is specified in the payload built by the service.  So I've overwritten `NewMessage.aifc` with the iOS default notification sound "note".

Reply:
Per external conversation, I think this is an improvement but it may be possible to respect the users "default message tone" choice here (but probably will never be possible to support their per-user choice).

Reply:
Yup. There's a checklist item in the card.